examples/src/main/res/layout/fragment_navigation_view.xml

```diff
@@ -0,0 +1,25 @@
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
A single `FrameLayout` with a `Button` is enough here, unless you plan to extend the layout. 
If you simplify the layout, then `NavigationView` will need to be added this way:
```kotlin
binding.navigationViewContainer.addView(navigationView, 0)
```

Reply:
Let's keep parity with the activity variant so that we can extend both in parallel.